### DISCUSSION TITLE
Trying to connect an ili9488 display to esp32-wrover-dev but the screen is completely black and never turns on.

### DISCUSSION THREAD

**Discussion by Defex111** [Bodmer/TFT_eSPI#2513] [Q&A - General] [answered]
![IMG-1590](https://user-images.githubusercontent.com/128281109/226160520-5b6cd990-7577-418d-a713-83410f7fac0f.jpg)
![IMG-1589](https://user-images.githubusercontent.com/128281109/226160521-3b13af04-4581-448a-9060-29ec692fbc92.jpg)
![IMG-1588](https://user-images.githubusercontent.com/128281109/226160523-8a2098f9-73f8-44c5-8722-2f183ae107ad.jpg)
![IMG-1587](https://user-images.githubusercontent.com/128281109/226160524-bee92fa5-0ac2-4c08-9eea-bc7c1b1a36fe.jpg)

in User_Setup.h i commented out the ILI9341 driver and uncommented the ILI9488 driver and i also did the following: // For ESP32 Dev board (only tested with ILI9341 display)
// The hardware SPI can be mapped to any pins

#define TFT_MISO 19
#define TFT_MOSI 23
#define TFT_SCLK 18
#define TFT_CS   15  // Chip select control pin
#define TFT_DC    2  // Data Command control pin
#define TFT_RST   4  // Reset pin (could connect to RST pin)
#define TFT_RST  -1  // Set TFT_RST to -1 if display RESET is connected to ESP32 board RST

the screen is completely off and im not sure what i am doing wrong

**Comment by Bodmer** [comment]
If the screen backlight LEDs are not turning on the you may need to connect the BL pin to 0V or 3.3V. The backlight LEDs must be on to see the screen image.

TFT_RST is defined twice, only define once  in this case it should be pin 4.

Run Read_User_Setup and check the serial output to verify the compiler is using the correct pins.

**Reply by Defex111** [reply]
Thank you! that has worked, my screen is now white but when i run the examples on arduino nothing changes, it just remains white

**Comment by Bodmer** [comment]
Run Read_User_Setup and check the serial output to verify the compiler is using the correct pins.

**Reply by Defex111** [reply]
do i run Read_User_Setup and look at the serial monitor?

**Reply by Defex111** [reply]
Display SPI frequency = 27.00
[/code]

[code]
TFT_eSPI ver = 2.5.0
Processor    = ESP32
Frequency    = 240MHz
Transactions = Yes
Interface    = SPI
Display driver = 9488
Display width  = 320
Display height = 480

MOSI    = GPIO 23
MISO    = GPIO 19
SCK     = GPIO 18
TFT_CS   = GPIO 15
TFT_DC   = GPIO 2

Font GLCD   loaded
Font 2      loaded
Font 4      loaded
Font 6      loaded
Font 7      loaded
Font 8      loaded
Smooth font enabled

**Comment by Bodmer** [comment, marked as answer]
TFT_RST is not defined correctly as it is missing in the report.

I notice also that the camera is connected to the pin used for the TFT so that may cause problems. Disconnect to camera to check if the display works.

**Reply by Defex111** [reply]
Thank you so much. i removed the camera and it turns out the RST was commented out in the user setup. It works now!
